vision/set_endpoint.rb

```diff
@@ -0,0 +1,39 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "uri"
+
+def set_endpoint
+  # [START vision_set_endpoint]
+  require "google/cloud/vision"
+
+  # Specify the service address at client construction.
+  image_annotator = Google::Cloud::Vision::ImageAnnotator.new service_address: "eu-vision.googleapis.com"
```

Comment:
Could you use variables rather than literal values, similar to how https://github.com/GoogleCloudPlatform/ruby-docs-samples/blob/master/speech/speech_samples.rb#L102 is set up?

Reply:
I think in this case we want to show a concrete service address in the doc page. @nnegrey is that correct?

Reply:
Yes, they product team asked for this exactly, there was a couple iterations on the samples before they approved these.

Reply:
updated to have the set variable statements in comment.  please take a look.

Reply:
Similar docs page this will be hosted on: https://cloud.google.com/natural-language/docs/locations